Demo/ImagePickerDemo/ImagePickerDemo/Cell.swift

```diff
@@ -0,0 +1,18 @@
+//
+//  Cell.swift
+//  ImagePickerDemo
+//
+//  Created by Khoa Pham on 11/8/15.
+//  Copyright © 2015 Ramon Gilabert Llop. All rights reserved.
+//
+
+import Foundation
+import UIKit
+
+class Cell : UICollectionViewCell {
```

Comment:
@vadymmarkov Nice catch. I just follow Apple (a little dogmatic of me :] ). For example

```swift
public class UIViewController : UIResponder, NSCoding, UIAppearanceContainer, UITraitEnvironment, UIContentContainer, UIFocusEnvironment {
```